src/goCover.ts

```diff
@@ -305,3 +309,19 @@ export function toggleCoverageCurrentPackage() {
 		});
 	});
 }
+
+export function isPartOfComment(e: vscode.TextDocumentChangeEvent): boolean {
+	let result = false
+	e.contentChanges.every(function(change, index){
+		let line = e.document.lineAt(change.range.start)
+		let text = e.document.getText(line.rangeIncludingLineBreak)
```

Comment:
What about the case when the range spans multiple lines and only the first line was part of a comment? In that case, `isPartOfComment` would return `true`